### PR TITLE
[office-js][office-js-preview] (Excel) Pulling latest Excel type definitions from CDN

### PR DESCRIPTION
**Pull request by alison-mk**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
Release URL: https://appsforoffice.microsoft.com/lib/1.1/hosted/office.debug.js
Beta URL: https://appsforoffice.microsoft.com/lib/beta/hosted/office.debug.js
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

### CODE REVIEW

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -24368,5615 +24368,5615 @@ declare namespace Excel {
...
+         */
         displayName?: string;
         /**
-        * Represents the count of rows which the card claims are in the array. A card may report a different number of rows than it actually has to display smaller amounts of preview data.
-        *
-        * @remarks
-        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
-        * @beta
-        */
+         * Represents the count of rows which the card claims are in the array. A card may report a different number of rows than it actually has to display smaller amounts of preview data.
```
Comment (Rick-Kirkham) [inline types/office-js-preview/index.d.ts:24477]: The 2nd sentence is ungrammatical. I can't tell what it means.

File: types/office-js-preview/index.d.ts [hunk 2]
```diff
@@ -24368,5615 +24368,5615 @@ declare namespace Excel {
...
+         */
         rowsToReport?: number;
         /**
-        * Represents the count of columns which the card claims are in the array. A card may report a different number of columns than it actually has to display smaller amounts of preview data.
-        *
-        * @remarks
-        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
-        * @beta
-        */
+         * Represents the count of columns which the card claims are in the array. A card may report a different number of columns than it actually has to display smaller amounts of preview data.
```
Comment (Rick-Kirkham) [inline types/office-js-preview/index.d.ts:24485]: The 2nd sentence is ungrammatical. I can't tell what it means.

File: types/office-js-preview/index.d.ts [hunk 3]
```diff
@@ -24368,5615 +24368,5615 @@ declare namespace Excel {
...
-        * Represents the value that would be returned by `Range.values` for a cell with this value.
-        * When accessed through a `valuesAsJson` property, this string value aligns with the en-US locale.
-        * When accessed through a `valuesAsJsonLocal` property, this string value aligns with the user's display locale.
-        *
-        * @remarks
-        * [Api set: ExcelApi 1.16]
-        */
+         * Represents the value that would be returned by `Range.values` for a cell with this value.
+         * When accessed through a `valuesAsJson` property, this string value aligns with the en-US locale.
+         * When accessed through a `valuesAsJsonLocal` property, this string value aligns with the user's display locale.
```
Comment (Rick-Kirkham) [inline types/office-js-preview/index.d.ts:24734]: I'm not a custom functions expert, but will devs understand what it means when a property is "accessed through" another property? (I don't.)

**Review by Rick-Kirkham — CHANGES_REQUESTED** [review]

File: types/office-js-preview/index.d.ts (hunk 3 above)
Comment (alison-mk) [reply types/office-js-preview/index.d.ts:24734]: This phrasing has been released to GA for over 2 years and we haven't gotten any questions about it, so I think it's ok.

File: types/office-js-preview/index.d.ts (hunk 1 above)
Comment (alison-mk) [reply types/office-js-preview/index.d.ts:24477]: Thanks for the review! I'll fix these.

**Review by Rick-Kirkham — APPROVED** [review]

File: types/office-js-preview/index.d.ts [hunk 4]
```diff
@@ -66121,11 +66111,180 @@ declare namespace Excel {
...
+         *
+         * @remarks
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         *
+         * @param cellAddress
+         * @param content
+         * @returns
+         */
+        add(cellAddress: Range | string, content: any): Excel.Note;
```
[the remark this answers is not in the document]
Comment (alison-mk) [reply types/office-js-preview/index.d.ts:66142]: Resolved here: https://github.com/OfficeDev/office-js-docs-reference/pull/2168

**Pull request merged**